README.md

```diff
@@ -319,7 +319,22 @@ We would be happy to merge your feature then.
 With this command, you can process your project with rector from docker:
 
 ```bash
-docker run -v $(pwd):/project rector/rector:latest process project/app --level symfony40 --dry-run
+docker run -v $(pwd):/project rector/rector:latest process project/src --level symfony40 --dry-run
```

Comment:
Please be so nice and edit, `project/src` to absolute path `/project/src` i think it was typo in original documentation.